tests/test_mock_controller.py

```diff
@@ -1182,3 +1198,26 @@ async def test_status(self) -> None:
         thcs_status = self.data[mtdome.LlcName.THCS.value]
         assert thcs_status["status"]["status"] == mtdome.LlcMotionState.CLOSED.name
         assert thcs_status["temperature"] == [0.0] * mtdome.mock_llc.NUM_THERMO_SENSORS
+
+    async def test_slow_network(self) -> None:
```

Comment:
It took me some time to figure out the point of this test. Please consider a doc string such as:
```
"""Test the slow_network feature of the mock controller."""
```
 It's probably not a big help, but it might keep the user from wondering if you are just testing that "it takes longer to read data if you pretend the network is slow", which is not interesting.

Reply:
Doc string added.